scripts/pre-commit-check-code-generation

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+set -e
+
+GO111MODULE=off go get golang.org/x/tools/cmd/goimports
```

Comment:
There might be a better way to do this. I added `GO111MODULE=off` so this wouldn't update go.mod / go.sum when run locally.